starknet_py/net/client.py

```diff
@@ -271,3 +271,19 @@ async def get_class_by_hash(self, class_hash: Hash) -> DeclaredContract:
         :param class_hash: Class hash
         :return: ContractClass object
         """
+
+    @abstractmethod
+    async def get_nonce(
```

Comment:
Let's call it `get_contract_nonce`. This avoids confusion in case of AccountClient.get_nonce which suggests it is account's nonce.